src/gurt/telemetry.c

```diff
@@ -1699,6 +1699,32 @@ d_tm_set_counter(struct d_tm_node_t *metric, uint64_t value)
 	d_tm_node_unlock(metric);
 }
 
+/**
+ * Decrement the given counter by the specified \a value
+ *
+ * \param[in]	metric	Pointer to the metric
+ * \param[in]	value	Increments the counter by this \a value
+ */
+void
+d_tm_dec_counter(struct d_tm_node_t *metric, uint64_t value)
```

Comment:
NB: This was not an accidental omission in the telemetry library. By convention, telemetry counter types are supposed to always increase. Downstream consumers are likely to be confused by a counter that decreases, as most consumers will interpret this as a counter rollover.

If you want to track a value that can fluctuate up or down, then a gauge is more appropriate.